example_cpp_smart_card_client_app/src/window.js

```diff
@@ -46,28 +46,29 @@ const logger = GSC.Logging.getLogger(
 
 goog.log.info(logger, 'The main window is created');
 
+//#revisitcode
```

Comment:
Ditto for not landing with these temporary comments. Let's keep the codebase clean at any moment of time.